### DISCUSSION TITLE
Module for delivery with Google Calendar and Google Contact

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#2000] [Extensions & Modules]
I am not a developer, I am a web designer.
for this reason a few years ago I funded and had 2 modules for Magento developed by a software house dietro mie diettive.

I am the owner of all rights

At the basis of the modules was the desire to take advantage of google services for booking home delivery services in the catering sector.

however, these modules do work and can be improved.

The modules are:
synchronization of customer data registered with google contact
Creation and synchronization of appointments with the choice of slot-time based on restaurant availability.

I am willing to donate these forms to the community as long as I can give indications on how they should work having been a takeaway restaurant manager.

Of course I am open to suggestions that can grow my basic idea.

if anyone is willing to take the development in hand let me know.

**Comment by Danieleeffe1** [comment]
Having not received any response, I realize that perhaps I need to post a demo link to give an idea of ​​what I would like to donate to see it developed and completed.

is meant to be viewed on mobile devices

[demo.menudigitale.eu](https://demo.menudigitale.eu)

**Comment by addison74** [comment]
As you can see from the OpenMage page there are 71 of us who follow it and receive immediate notifications. My recommendation is to add it to a public repository, as long as you have all the property rights. After you make it public you announce it here. Thus it will be found by other people who could help its development.
